ui/component/notification/index.js

```diff
@@ -1,8 +1,14 @@
 import { connect } from 'react-redux';
```

Comment:
Second level reply is a bit weird. The GUI remains stuck at the following, with SuperChat appearing.  When I pressed Cancel, I noticed a ton of comment was sent.

![image](https://user-images.githubusercontent.com/64950861/129561235-cf3ee253-add8-4f82-b352-a036d9532f92.png)